plumber/cli_batch.go

```diff
@@ -42,3 +49,27 @@ func (p *Plumber) HandleBatchCmd() error {
 		return fmt.Errorf("unrecognized command: %s", p.CLIOptions.Global.XFullCommand)
 	}
 }
+
+// logMetricBatch sends a metric with only the command name.
+// We don't want to ingest flags for privacy
+func (p *Plumber) logMetricBatch(cmd string) {
+	var cmdName string
+
+	parts := strings.Split(cmd, " ")
+	if len(parts) > 2 {
+		cmdName = strings.Join(parts[1:2], " ")
+	} else if len(parts) > 1 {
+		cmdName = parts[1]
+	}
+
+	p.PostHog.Enqueue(posthog.Capture{
+		DistinctId: p.Config.PersistentConfig.PlumberID,
+		Event:      "batch",
+		Properties: map[string]interface{}{
+			"version":         options.VERSION,
+			"command":         cmdName,
+			"os":              runtime.GOOS,
+			"$geoip_disabled": true,
```

Comment:
Also, if read or write - should include if encode-type or decode-type is specified (and what type). That way we know the popularity of the different formats.